e2e/tests/adminManagesOrders_test.js

```diff
@@ -45,20 +46,25 @@ Scenario('Create C21 blank order', async ({I, caseViewPage, manageOrdersEventPag
   await I.goToNextPage();
   await manageOrdersEventPage.selectChildren(manageOrdersEventPage.section3.allChildren.options.select, [0]);
   await I.goToNextPage();
+  await manageOrdersEventPage.enterTitle(orderTitle);
   await manageOrdersEventPage.enterDirections('some text');
   await I.goToNextPage();
   await manageOrdersEventPage.checkPreview();
   await I.completeEvent('Save and continue');
   I.seeEventSubmissionConfirmation(config.administrationActions.manageOrders);
-  assertOrder(I, caseViewPage, 2, 'C21 - Blank order', approvalDate, allocatedJudge, 'Timothy Jones');
+  assertOrder(I, caseViewPage, 2, 'C21 - Blank order', 'some title', approvalDate, allocatedJudge, 'Timothy Jones');
 });
 
-function assertOrder(I, caseViewPage, orderIndex, orderType, approvalDate, judge, children) {
+function assertOrder(I, caseViewPage, orderIndex, orderType, orderTitle, approvalDate, judge, children) {
   const orderElement = `Order ${orderIndex}`;
   caseViewPage.selectTab(caseViewPage.tabs.orders);
   I.seeInTab([orderElement, 'Type of order'], orderType);
   I.seeInTab([orderElement, 'Approval date'], dateFormat(dateToString(approvalDate), 'd mmm yyyy'));
   I.seeInTab([orderElement, 'Judge and Justices\' Legal Adviser', 'Judge or magistrate\'s title'], judge.title);
   I.seeInTab([orderElement, 'Judge and Justices\' Legal Adviser', 'Last name'], judge.name);
   I.seeInTab([orderElement, 'Children'], children);
+
+  if(orderTitle != null){
```

Comment:
should probs be !==

Reply:
if we test against undefined then in the C32 test I think we can call it with
```
  assertOrder(I, caseViewPage, 1, 'C32 - Care order', , approvalDate, allocatedJudge, 'Timothy Jones');
```
although it might require setting a default value but that probs looks better than a random null in the method call